code/game/objects/items/weapons/shields.dm

```diff
@@ -39,6 +39,16 @@
 	icon_state = "roman_shield"
 	item_state = "roman_shield"
 
+/obj/item/weapon/shield/riot/buckler
+	name = "wooden buckler"
+	desc = "A medieval wooden buckler."
+	icon_state = "buckler"
+	item_state = "buckler"
+	materials = list()
+	slot_flags = null
+	force = 15

```

Comment:
I was going to say a meme but I forgot the exact words but 15 force isn't that much for bashing heads in